consent/manager_test_helpers.go

```diff
@@ -41,7 +41,7 @@ import (
 
 func MockConsentRequest(key string, remember bool, rememberFor int, hasError bool, skip bool, authAt bool) (c *ConsentRequest, h *HandledConsentRequest) {
```

Comment:
Is there a particular reason for changing `challenge` to `consent-challenge`, `fk-login-challenge` to `login-challenge`, and `time.Minute` to `time.Hour`?

Reply:
(1/n) regarding the RequestedAt 'time.Hour' -> 'time.Minute':
I changed this in two places, 1) in the mocked consent request, and 2) in the mocked login request. In the original tests, these values were ignored and the tests passed no matter the time, because the value that was actually used (in tests, not in application code) was the one from HandledConsentRequest, which always was 'time.Minute'. There is only one RequestedAt column now, and it is assigned from the LoginRequest as opposed to HandledConsentRequest (this has always been the case in application code). It follows that changing RequestedAt in the mocked consent request is unnecessary, so I reverted this change in the latest commit. The 'time.Hour' -> 'time.Minute' in MockAuthRequest is necessary for the above reasons.

Reply:
Thank you!